modules/priceFloors.js

```diff
@@ -249,8 +249,22 @@ export function getFloor(requestParams = {currency: 'USD', mediaType: '*', size:
 
   // if cpmAdjustment flag is true and we have a valid floor then run the adjustment on it
   if (floorData.enforcement.bidAdjustment && floorInfo.matchingFloor) {
-    let cpmAdjustment = getBiddersCpmAdjustment(bidRequest.bidder, floorInfo.matchingFloor);
-    floorInfo.matchingFloor = cpmAdjustment ? calculateAdjustedFloor(floorInfo.matchingFloor, cpmAdjustment) : floorInfo.matchingFloor;
+    // For cpm adjustments, they may use the "bid response" object now. So we should put as much info in it as we can
+    // But since this is at bidRequest time, we may not have everything! So put entire bidRequest and overwrite a couple things to be safe
+    const fakeBid = {
```

Comment:
I think for the inverse it's better to just pass in the bidRequest no? since we never have a bid we don't need to fake it. This would also be the first exposure to the internal `getFloor` params we'd give to the publisher, I'm not sure that they'd help more than the request.

Reply:
well in the case that the bidRequest has both banner and video

But the bidder is asking for floor specific to banner or video,

I think it is best to expose that information to them right?

Same with sizes, if there are 5 sizes, and a bidder loops through all and asks for the specific floor for that size I guess?

But yeah only makes sense if the adjustment / inverse function relys on the attribute.

Which I think bidder code, and mediaType are definitely used in adjustment functions.

Reply:
I'm unconvinced, I think most bidders ask for star, so you still can't provide the same info you'd have in the response - and I think we are more likely to get confusion from `mediaType = '*'` than precision from when it is specified. I would be OK with passing them in separately such as `(bidRequest, floorRequestParams)`, except that the latter is still an internal API and I'd prefer to not lock that in prematurely.

Reply:
ok one last ask for input from @patmmccann 

In your scenario where your regular bid adjustment function only subtracts `0.18` if the mediaType is video,

In the inverse function, if we pass you the bidRequest, is that enough?

The only way it wouldn't would be if bidRequest.mediaTypes had both banner and video right?

Also @dgirardi My thought was that the input params to the `bidCpmAdjustment` function and the `inverseBidAdjustment` functions should be aligned so it is easy for pubs to utilize them?

Or you think it is ok to just have them different.

I agree regarding exposing the `floorRequestParams` fwiw

Reply:
Yeah seems right to me